client/components/single-pothole/CommentList.js

```diff
@@ -1,35 +1,47 @@
 import React, { Component } from 'react';
 import { Comment } from 'semantic-ui-react';
+import { connect } from 'react-redux';
 
 class CommentList extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-      allComments: props.allComments[0],
-    };
+    this.state = {};
   }
 
   render() {
-    return this.state.allComments.map(comment => {
-      return (
-        <Comment.Group key={comment.id}>
-          <Comment>
-            <Comment.Avatar
-              as="a"
-              src="https://www.cxservice360.com/wp-content/uploads/2017/09/Avatar.png"
-            />
-            <Comment.Content>
-              <Comment.Author>{comment.user.firstName}</Comment.Author>
-              <Comment.Metadata>
-                <div>2 days ago</div>
-              </Comment.Metadata>
-              <Comment.Text>{comment.text}</Comment.Text>
-            </Comment.Content>
-          </Comment>
-        </Comment.Group>
-      );
-    });
+    if (this.props.allComments.length) {
+      return this.props.allComments.map(comment => {
+        return (
+          <Comment.Group key={comment.id}>
+            <Comment>
+              <Comment.Avatar
+                as="a"
+                src="https://www.cxservice360.com/wp-content/uploads/2017/09/Avatar.png"
+              />
+              <Comment.Content>
+                <Comment.Author>{comment.user.firstName}</Comment.Author>
+                <Comment.Metadata>
+                  <div>2 days ago</div>
+                </Comment.Metadata>
+                <Comment.Text>{comment.text}</Comment.Text>
+              </Comment.Content>
+            </Comment>
+          </Comment.Group>
+        );
+      });
+    } else {
+      return <div />;
+    }
   }
 }
 
-export default CommentList;
+const mapState = state => {
```

Comment:
Why do we have to also map this component to state? Did it not work when mapping the parent component (i.e., SinglePotholeComments?

Reply:
I cleaned up couple things but let me know if you wanna go over something before merging to master.

Reply:
I pulled the latest and the screen does not update with the newly added comment

Reply:
And when I refresh I get a sequelize error for and undefined integer syntax

Reply:
When u get a chance please take a look to the new changes. Thanks!!!